src/util/real_algebraic_number_poly_imp.cpp

```diff
@@ -0,0 +1,147 @@
+#include "cvc4autoconfig.h"
```

Comment:
Copyright header missing, you can auto-add it with `./contrib/update-copyright.pl <file>`.